lib/plugin.js

```diff
@@ -10,7 +10,12 @@ class VueLoaderPlugin {
     if (compiler.hooks) {
       // webpack 4
       compiler.hooks.compilation.tap(id, compilation => {
-        compilation.hooks.normalModuleLoader.tap(id, loaderContext => {
+        let { normalModuleLoader } = compilation.hooks;
+        if (!normalModuleLoader) {
```

Comment:
`compilation.hooks.normalModuleLoader` still exists in webpack 5, just deprecated, and a `get` operation will trigger the warning, so it's better to let the new `getCompilationHooks` method take a higher priority.